### ISSUE TITLE
AutoComplete: DefaultCommand not triggered when Popup is opened

### ISSUE THREAD

**Issue by fcjm** [primefaces/primefaces#10847]
### Describe the bug

We have a use-case, where users can search our platform for a specific string. For that we use the autoComplete component with "autoHighlight=false". You can either enter any string you want and press the search button next to the autoComplete, or you select one of the suggestions, where a redirect to a specific resource gets triggered.

From the user perspective you would expect to perform the action of the button next to the autoComplete if you press Enter after entering your search term. For that we use a defaultCommand which triggers the action of the button. 

However, at the moment this doesn't seem to work as soon as the popup with the suggestions opens. If you close it with Escape and then hit Enter again, everything works fine.

### Reproducer

[reproducer.zip](https://github.com/primefaces/primefaces/files/12999738/reproducer.zip)

1. enter any string into the autoComplete
2. wait for the suggestions to show up
3. hit Enter -> nothing happens
4. hit Escape
5. hit Enter again -> button action gets performed

### Expected behavior

DefaultCommand should also work when the suggestion popup is opened.

### PrimeFaces edition

Community

### PrimeFaces version

13.0.2

### JSF implementation

Mojarra

### JSF version

2.3

### Java version

11

**Comment by fcjm** [comment]
Thank you for the response. 

I agree that, if you interact with the popup, the currently highlighted item should be selected and afterwards the popup closed. I think that only applies when the visible focus is within the popup though. When I use the autoComplete with "autoHighlight=false" the visible focus never actually gets put into the popup. So for the user it should more or less act like a normal text inputfield.

I have found [this example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/#kbd_label) from WAI.

There they also distinguish between keyboard navigation on the textbox, and within the popup. Their solution is to close the popup when the visible focus is on the textbox and Enter is being pressed. That would also suffice for our scenario, even if it means that you have to hit enter twice in order to submit the form. 

In my opinion it would be ideal if the popup closes and directly afterwards the default action gets executed as well. That way the autoComplete would act like any other text inputfield.

Please let me know what you think.

**Comment by melloware** [comment]
Have you looked into a PR? Would also like @christophs78 thoughts since he and his co-worker just recently added the ARIA and Accessibility to AutoComplete.
